build.sbt

```diff
@@ -6,10 +6,16 @@ val Library = new {
     val circe = "0.7.1"
     val refined = "0.8.0"
     val scalaTest = "3.0.0"
+    val finagle = "6.43.0"
```

Comment:
these dependencies are used by single module, please move them into the `build.sbt` inside the module